docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,32 @@
+# Constructing a PR to add an Operator
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via OLM
+1. Manual install
+
+## Install an operator via OLM
+
+Operator Lifecycle Manager (OLM) makes it easy to install and manage Operators on OpenShift clusters.
+To install an Operator via OLM, you must make a PR that does the following:
+
```

Comment:
Assume more hand holding, give step by step instructions starting with: 
1. clone apps repo <give link> 
2. navigate to <subscription-folder> 
3. etc.